tests/runtest.py

```diff
@@ -1000,6 +1068,7 @@ def parse_argument():
     patch_size = {
         'x86_64'  : 5,
         'aarch64' : 2,
+        'riscv' : 2,
```

Comment:
@gichoel I think same with aarch64. We need to double checking.

Reply:
As you may already know, the RISC-V architecture does not yet support `dynamic tracing`. Therefore, related code must be added after introducing `Dynamic Tracing` in the RISC-V architecture.

For the above reasons and to reduce confusion, I think you should remove `riscv` from the `patch_size` variable in that code, so that the string `fpatchable-function-entry not support on current platform` is printed and the program exits.

Additionally, although I'm not sure yet, there is a commit that forces RISC-V kernels to use `-fpatchable-function-entry=8`, so you should check if you can use the same value as `aarch64`.
ref. https://git.kernel.org/torvalds/c/afc76b8b8011

Reply:
Thanks for the review. I will include both @MichelleJin12 @gichoel feedback. :)